server/src/ai/ai.router.ts

```diff
@@ -1,28 +1,33 @@
 import express from 'express';
 import { CourseIdRequestType } from '../course/course.type';
-import { makeSummary, getCoursesWithMinReviews, getReviewsForSummary } from './ai.functions';
+import { getCoursesWithMinReviews, getReviewsForSummary, generateTags } from './ai.functions';
 
 const aiRouter = express.Router();
 
 aiRouter.use(express.json());
 
-/** Reachable at POST /api/ai/summarizeReviews
+/** Reachable at POST /api/ai/text/tags
  * @body a block of text containing all reviews from a course
- * returns a summary created by OpenAI
+ * returns a dictionary containing the summary and tags created by OpenAI
 */
-aiRouter.post('/text/summary', async (req, res) => {
+aiRouter.post('/text/summary/tags', async (req, res) => {
   try {
-    if (!req.body.text) {
+    const { text } = req.body;
+    if (!text) {
       return res.status(400).json({ error: 'No text provided' });
     }
-    const summary = await makeSummary(req.body.text);
-    res.status(200).json({ summary });
+    const summaryAndTags = await generateTags(text);
+    res.status(200).json({ summaryAndTags });
   } catch (err) {
     console.error(err);
     res.status(500).json({ error: 'Internal Server Error' });
   }
 });
 
+/** Reachable at POST /api/ai/get/text
+ * @body a course ID
+ * returns a block of text containing all reviews from that course
+*/
 aiRouter.post('/get/text', async (req, res) => {
```

Comment:
also maybe change this to 
`api/ai/get-course-reviews` or `api/ai/get-course-reviews-text`?